as_perimenoun_oi_gynaikes.ass

```diff
@@ -1066,11 +1067,11 @@ Dialogue: 0,1:16:24.56,1:16:27.54,Default,,0,0,0,,My wife didn't do anything out
 Dialogue: 0,1:16:27.54,1:16:29.79,Default,,0,0,0,,And she calls me two days ago and she was acting all nice...
 Dialogue: 0,1:16:29.79,1:16:31.26,Default,,0,0,0,,"Panos, when are you coming?"
 Dialogue: 0,1:16:31.26,1:16:33.01,Default,,0,0,0,,"We've been alone for a month."
-Dialogue: 0,1:16:33.01,1:16:35.51,Default,,0,0,0,,Like she had a clue.
+Dialogue: 0,1:16:33.01,1:16:35.51,Default,,0,0,0,,Like they had a clue.
 Dialogue: 0,1:16:35.51,1:16:36.94,Default,,0,0,0,,Doesn't matter.
-Dialogue: 0,1:16:36.94,1:16:38.94,Default,,0,0,0,,Are they having a good time now?
+Dialogue: 0,1:16:36.94,1:16:38.94,Default,,0,0,0,,Aren't they triggered now?

```

Comment:
Triggered? Why triggered?

Reply:
Φτιαγμένος in Greek does not mean always happy/high. I believe that in this contexts refers to the fact that hsubands will hear nagging when they go to Thasos because they are late so many days. So, they are 'triggered'/waiting for them. For sure wives are not happy.

Reply:
Hmmm. I can't say that I disagree. Let's change it then.